### PR TITLE
Bl orient 812

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient812.xml [hunk 1]
```diff
@@ -49,50 +132,120 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"></locus>
+                              <desc>
+                                A rude table of the signs of Zodiac.
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient812.xml:168]: @smaugustine , can you help us? I always wonder if Wright's "rude" (which probably means what is described as "crude"  for instance in Ethio-SPaRe descriptions) would sound strange to modern English speakers and should be substituted by another word in these descriptions.

File: LondonBritishLibrary/orient/BLorient812.xml [hunk 2]
```diff
@@ -30,17 +31,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <msItem xml:id="ms_i1.4.1">
+                              <title type="complete" xml:lang="en">The list of the patriarchs
+                              of Alexandria</title>
+                              <explicit xml:lang="gez"><locus target="180v"></locus>
+                              ፺ወ፯ <persName ref="PRS4381Gabriel">አባ፡ ገብርኤል፡</persName> ዘሀሎ፡ በዘመነ፡
+                              <persName ref="PRS8550sardaDe">ንጉሥ፡ ሠርፀ፡ ድንግል።</persName>
+                            <sic>፺ወ፰</sic> <persName ref="PRS5710JohnXV">አባ፡ ዮሐንስ።</persName>
+                            <sic>፺ወ፱</sic> <persName ref="PRS6905Matthew">አባ፡ ማቴዎስ።</persName>
+                            <sic>፻</sic> <persName ref="PRS6752Mark">አባ፡ ማርቆስ።</persName>
+                              <!-- the numbers are not conform to the traditional system, Mark V is
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient812.xml:75]: I think that his could be changed into a note: if you noticed this, others also might find it interesting.

File: LondonBritishLibrary/orient/BLorient812.xml [hunk 3]
```diff
@@ -30,17 +31,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            skipped, as far as I understand-->
+                            </explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4.2">
+                              <title type="complete" xml:lang="en">The Seven Wonders of the World</title>
+                              <textLang mainLang="gez"></textLang>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4.3">
+                              <locus target="#181r"></locus>
+                              <title type="complete" xml:lang="en">The list of ʾabunās of Ethiopia</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient812.xml:85]: ʾabuna, not ʿabunā?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**